python/paddle/trainer_config_helpers/layers.py

```diff
@@ -1249,6 +1253,101 @@ def detection_output_layer(input_loc,
         name, LayerType.DETECTION_OUTPUT_LAYER, parents=parents, size=size)
 
 
+@wrap_name_default("rcnn_loss")
+def rcnn_loss_layer(rois,
+                    input_loc,
+                    input_conf,
+                    loss_ratio,
+                    num_classes,
+                    background_id=0,
+                    name=None):
```

Comment:
it's better to add some comments for the difference between `rnn_loss` and `multibox_loss`, if you understand.